### PR TITLE
Added Eversticky

### PR DESCRIPTION
**Pull request by Xmetalfanx**
Closes https://github.com/wimpysworld/deb-get/issues/230

Tested and works locally

### CODE REVIEW

File: deb-get [hunk 1]
```diff
@@ -2413,6 +2413,17 @@ function deb_mullvad-vpn() {
     SUMMARY="VPN Service based in Sweden that does not collect user data."
 }
 
+function deb_eversticky() {
+    get_github_releases "https://api.github.com/repos/itsmejoeeey/eversticky/releases/latest"
+    if [ "${ACTION}" != "prettylist" ]; then
+        URL="$(grep "browser_download_url.*.deb" "${CACHE_DIR}/${APP}.json" | head -n1 | cut -d'"' -f4)"
+        VERSION_PUBLISHED="$(echo "${URL}" | cut -d'/' -f8)"
```
Comment (natanjunges) [inline deb-get:2420]: This should have `tr -d v` at the end.

**Review by natanjunges — CHANGES_REQUESTED** [review]

**Comment by Xmetalfanx** [comment]
updated again ... so sorry about these issues ... I really should have checked some of the code i used that I assumed worked.   (was part of a bigger PR that addressed different issues all at once) ... that is one of the reasons i decided to do PR by PR in case some were ok but others had issues.

I am going back over my mistakes and will double check things before submitting PR's from now on

**Comment by natanjunges** [comment]
No problem, just ensuring the quality of the PRs.

**Review by flexiondotorg — CHANGES_REQUESTED** [review]
Please drop the Virtualbox change in README.md, rebase and squash the commits.

**Comment by natanjunges** [comment]
The format of package definitions has changed. Please read [01-main/CONTRIBUTING](https://github.com/wimpysworld/deb-get/blob/main/01-main/CONTRIBUTING.md) and [EXTREPO](https://github.com/wimpysworld/deb-get/blob/main/EXTREPO.md), update your implementation and submit a new PR.

**Pull request closed without merging**